docs/development/client.rst

```diff
@@ -14,7 +14,7 @@ The source code, and related issues are `hosted on GitHub <https://github.com/fr
 Developer Setup
 ---------------
 
-You may find developer setup instructions in the `SecureDrop Client README <https://github.com/freedomofpress/securedrop-client/blob/master/README.md>`_.
+You may find developer setup instructions in the `SecureDrop Client README <https://github.com/freedomofpress/securedrop-client/blob/main/README.md>`_.
```

Comment:
Pointed out by @maeve-fpf earlier in the branch renames: if you use HEAD in the URLs, GitHub will automatically show you whatever the default branch has, e.g. https://github.com/freedomofpress/securedrop-client/blob/HEAD/README.md Sharing in case it's useful in other contexts, not requesting a change here.

Reply:
Thanks! I'm actually gonna reissue this PR with a properly named branch to unstick CI, so I'll make this change regardless.